src/api/ecom-api.tsx

```diff
@@ -88,23 +88,23 @@ function createApi(): EcomAPI {
                 }
                 return successResponse(product);
             } catch (e) {
-                return failureResponse(EcomApiErrorCodes.GetProductFailure, getErrorMessage(e));
+                return failureResponse(EcomApiErrorCodes.GetProductFailure, getErrorMessage(e) ?? 'Unknown error');
             }
         },
         async getCart() {
             try {
                 const currentCart = await wixClient.currentCart.getCurrentCart();
                 return successResponse(currentCart);
             } catch (e) {
-                return failureResponse(EcomApiErrorCodes.GetCartFailure, getErrorMessage(e));
+                return failureResponse(EcomApiErrorCodes.GetCartFailure, getErrorMessage(e) ?? 'Unknown error');
```

Comment:
just add 'Unknown error' inside `failureResponse` helper function

Reply:
or maybe not default to `Unknown error` at all, due to the same reason @alisey mentioned before.. to allow caller provide his own default message.

Reply:
Said this in another comment, but no one in their right mind should be doing things like `throw ''` / `throw undefined` / `throw { toString: () => ''}`, and it's not our job to work around things like that. If the error message is crap, then so be it and we shouldn't attempt to correct it post factum.